README.md

```diff
@@ -262,7 +262,239 @@ export default NuxtAuthHandler({
+        >
+          <ul class="flex justify-evenly my-auto">
+            <li class="my-auto mr-6">Link</li>
+            <li class="my-auto mr-6">Link</li>
+            <li>
+              <button
+                @click="signOut"
+                class="px-2 py-1 bg-orange-500 hover:bg-orange-600 text-white rounded-xl"
+              >
+                Sign Out
+              </button>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+
+    <div class="container mx-auto my-5 p-5">
+      <div class="md:flex no-wrap md:-mx-2">
+        <div class="w-full md:w-3/12 md:mx-2">
+          <div class="bg-white rounded-b-xl p-3 border-t-4 border-green-400">
+            <h2 class="text-gray-900 font-bold text-xl leading-8 my-1">
+              Welcome
+            </h2>
+            <p class="text-gray-700 py-4">
+              This is your protected dashboard area.
+            </p>
+          </div>
+        </div>
+
+        <div class="w-full md:w-9/12 mx-2 h-64">
+          <div class="bg-white p-3 shadow-sm rounded-xl">
+            <div
+              class="flex items-center space-x-2 font-semibold text-gray-900 leading-8"
+            >
+              <span class="tracking-wide">Protected Resource</span>
+            </div>
+
+            <div class="text-gray-700">
+              <div class="grid md:grid-cols-1 text-sm mt-4">
+                <div class="grid grid-cols-2">
+                  <div class="px-4 py-2 font-semibold">Data</div>
+                  <div class="px-4 py-2">{{ data }}</div>
+                </div>
+              </div>
+            </div>
+
+            <div class="flex">
+              <button
+                class="block w-full text-blue-800 text-sm font-semibold rounded-lg hover:bg-gray-100 focus:outline-none focus:shadow-outline focus:bg-gray-100 hover:shadow-xs p-3 my-4"
+              >
+                Refresh data
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+```
+For this example we won't focussing on the design of the page but we will be focussing on how to access restricted Strapi resources through that page.
+
+Within the `<script setup>` section of the above code, take notice of the following lines.
+```ts
+// In this section of the Nuxt 3 application we need the session to be required, so we've set the required field to true.
+const { signOut } = await useSession({ required: true })
+
+// We need to take the existing headers from the current session and pass them on to the admin endpoint we are creating.
+const headers = useRequestHeaders(['cookie'])
+const { data, pending, error, refresh } = await useFetch(`/api/admin/data`, { headers: { cookie: headers.cookie } });
+```
+
+We're going to call this admin API endpoint, but first we have to create the endpoint in Nitro.
+
+###### Creating a nitro endpoint for authenticated calls
```

Comment:
Same as above, same reasoning: Please remove

For future contributions I can recommend to:
- stay narrow in the scope you're adding
- if you're uncertain whether a PR is the right  place for the thing you want to do: Open an issue first

See https://github.com/sidebase/nuxt-auth#getting-the-jwt-token.